interfaces/i18n.js

```diff
@@ -0,0 +1,14 @@
+type chrome$LanguageCode = string;
+
+type chrome$i18n = {
+  getAcceptLanguages(callback: (Array<chrome$LanguageCode>) => void): void,
+  getMessage(messageName: string, ...substitutions?: Array<*>): ?string,
```

Comment:
`substitutions` is just an array parameter, not rest parameters https://developer.chrome.com/extensions/i18n#method-getMessage

Reply:
Nice catch, thanks.

I'm using the API, and I missed the note that it wasn't rest params. It actually works with a single string, but more than one requires an Array. The examples even mention the single string case:

https://developer.chrome.com/apps/i18n#examples-getMessage

It seems best to document the example and the actual API even though their documentation doesn't agree.

What do you think?